coderd/notifications/notifications_test.go

```diff
@@ -490,12 +491,15 @@ func TestRetries(t *testing.T) {
 
 	mgr.Run(ctx)
 
-	// THEN: we expect to see all but the final attempts failing
+	// the number of tries is equal to the number of messages times the number of attempts
+	// times 2 as the Enqueue method pushes into both the defined dispatch method and inbox
+	nbTries := msgCount * maxAttempts * 2
```

Comment:
If we conditionalize when inbox is used in the future, this will become brittle and it won't be easily understood why; this is why magic numbers are to be avoided at ~all~ reasonable costs.